js/index.js

```diff
@@ -0,0 +1,69 @@
+const books = [{}];
+
+class Books {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  addBook(book) {
+    const booksDiv = document.getElementById('list');
+    const div = document.createElement('div');
+    div.setAttribute('class', 'book-info');
+    div.setAttribute('id', 'book-info');
+    booksDiv.append(div);
+    const title = document.createElement('h5');
+    title.setAttribute('class', 'title');
+    title.innerHTML = this.title;
+    div.append(title);
+    const author = document.createElement('p');
+    author.setAttribute('class', 'author');
+    author.innerHTML = this.author;
+    div.append(author);
+    const button = document.createElement('button');
+    button.setAttribute('class', 'removebook');
+    button.addEventListener('click', (e) => {
+      e.stopPropagation();
+      const element = document.getElementById('book-info');
+      element.remove();
+    });
+    button.innerHTML = 'Remove';
```

Comment:
- [ ] removing the book doesn't actually remove the book from the `books` array.

Reply:
Working on it.

Reply:
Fixed as advised